recipes-devtools/abseil-cpp/abseil-cpp_git.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
```

Comment:
I you have the error message lying around, it would be a useful addition to the commit message.

Reply:
Added the errors as shown in the systemd journal.